packages/app/src/components/swap/exit.tsx

```diff
@@ -305,7 +306,16 @@ const LPTokensToRemoveInputField: FC<{
   );
 
   return (
-    <FormField label="LP Tokens to remove">
+    <FormField
+      label="LP Tokens to remove"
+      name="lpTokenToRemove"
+      validate={() => {
+        if (isPositiveAndSmallerThanMax(lpSharesInput, lpTokenBalance)) {
+          return { message: "Please provide an available amount", status: "error" };
+        }
+        return undefined;
+      }}
```

Comment:
Code style suggestion

```suggestion
      validate={() => 
       isPositiveAndSmallerThanMax(lpSharesInput, lpTokenBalance) ?  { message: "Please provide an available amount", status: "error" } :  undefined;
      }
```